src/rust/Cargo.toml

```
@@ -12,6 +12,7 @@ panic = "unwind"
 
 [dependencies]
 ciborium = "0.2"
+either = "1.13.0"
```

```suggestion
either = "1.11"
```

See https://github.com/pola-rs/polars/blob/fb28c07c612f89398436e597cf842b99037bd58d/Cargo.toml#L39